CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## [Unreleased]
 
+- Bump Rust to 1.86.0
```

Comment:
Here we can ass a sentence to explain the consequences wrt. compatibility. I think we should then use 0.17.0 as the next version to highlight the break for CW2 users.